demo/ez-toolbar.html

```diff
@@ -20,7 +20,20 @@
       <h3>Basic ez-toolbar demo</h3>
       <demo-snippet>
         <template>
-          <ez-toolbar></ez-toolbar>
+          <ez-toolbar>
+            <main>
+              <p>I'm a toolbar</p>
+            </main>
+          </ez-toolbar>
+          <button onclick="
+            if (document.querySelector('ez-toolbar').hasAttribute('visible')) {
```

Comment:
please use properties as the main *public API* for element. So to toggle the visibility of the toolbar, you are supposed to the `.visible` property.
So this can be greatly simplified to something like:

```js
document.querySelector('ez-toolbar').visible = !document.querySelector('ez-toolbar').visible
```